impeller/renderer/backend/vulkan/allocator_vk.h

```diff
@@ -27,6 +27,10 @@ class AllocatorVK final : public Allocator {
 
   fml::RefPtr<vulkan::VulkanProcTable> vk_;
   VmaAllocator allocator_ = {};
+  VmaPool raster_buffer_pool_ = {};
```

Comment:
When we go forward with this we should have a docstring for the different pools.  This is complicated enough that making it clear is helpful.